services/management/enclave/src/service.rs

```diff
@@ -938,7 +939,11 @@ impl TeaclaveManagementService {
             .connect()
             .await
             .map_err(|e| anyhow!("Failed to connect to storage service, {:?}", e))?;
-        let storage_client = Arc::new(Mutex::new(TeaclaveStorageClient::new(channel)));
+        let storage_client = Arc::new(Mutex::new(
+            TeaclaveStorageClient::new(channel)
```

Comment:
Calling those two max functions with the same parameter seems not smart. How about putting the default configuration in the constructor?